public/css/style.css

```diff
@@ -380,3 +381,21 @@ textarea {
   ;
   text-align: center;
 }
+
+.language_header {
+  width: 100%;
+  padding-left: 20px;
+  padding-top: 10px;
+  font-size: 1.25em;
+}
+
+.select_language_header {
+  float: left;
+  width: 20%;
+}
+
+.language_options {
+  float: left;
+  margin-top: 0;
+  width: 80%;
+}
```

Comment:
the underscores should be `-`'s.
The `padding-left` and `padding-right` should be rem values.
Should use CSS Flexbox here instead of floats and percentage widths.

Reply:
After looking at the styles more closely in the React component, the class names should use the BEM format like below:

```css
.language-selector {...}
.language-selector--header {...}
.language-selector--options {...}
```

Every Component should match the respective BEM class names like the above.

Reply:
Updated.